frontend/packages/rhoas-plugin/src/components/service-list/ServiceInstance.tsx

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable prettier/prettier */
```

Comment:
Is it necessary?

Reply:
good catch. we should never disable prettier